data/assets/scene/solarsystem/heliosphere/bastille_day/magnetogram_textures.asset

```diff
@@ -58,3 +58,15 @@ asset.onDeinitialize(function ()
   end
   openspace.action.removeAction(switch_color_layer.Identifier)
 end)
+
+
+asset.meta = {
+  Name = "Bastille Day magnetogram textures",
+  Version = "1.0",
+  Description = [[ This asset adds multiple magnetogram textures to the sun. In addition
```

Comment:
very nitpicky, but I would capitalize "the Sun", as to comply with [NASA guidelines](https://history.nasa.gov/styleguide.html)